beacon-chain/core/blocks/block.go

```diff
@@ -37,43 +37,6 @@ func NewGenesisBlock(stateRoot []byte) *pb.BeaconBlock {
 	return block
 }
 
-// BlockRoot returns the block root stored in the BeaconState for a given slot.
-// It returns an error if the requested block root is not within the BeaconState.
-// Spec pseudocode definition:
-// 	def get_block_root(state: BeaconState, slot: int) -> Hash32:
-//		"""
-//		returns the block root at a recent ``slot``.
-//		"""
-//		assert state.slot <= slot + SLOTS_PER_HISTORICAL_ROOT
-//		assert slot < state.slote
-//		return state.latest_block_roots[slot % SLOTS_PER_HISTORICAL_ROOT]
-func BlockRoot(state *pb.BeaconState, slot uint64) ([]byte, error) {
```

Comment:
Why delete these methods?

Reply:
It's no longer used, the new method is here:
https://github.com/prysmaticlabs/prysm/blob/spec-v0.6/beacon-chain/core/helpers/block.go#L42